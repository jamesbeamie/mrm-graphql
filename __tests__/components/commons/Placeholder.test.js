import React from 'react';
import { shallow } from 'enzyme';
import PlaceHolder from '../../../src/components/commons/Placeholder';

describe('PlaceHolder componenent', () => {
  it('should renders correctly', () => {
    expect(shallow(<PlaceHolder />)).toMatchSnapshot();
  });
});
