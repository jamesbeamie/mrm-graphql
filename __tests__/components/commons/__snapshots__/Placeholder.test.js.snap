// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceHolder componenent should renders correctly 1`] = `
<div>
  Placeholder
</div>
`;
